### PR TITLE
Backport logout fix to 3.10.x

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

Switch to GitHub Actions

medic/cht-core#7169

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
hey @latin-panda 

Could you please give my GHA build a sanity check? The other commits are backports of the #7169 fix. 
Thanks!

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    - name: Grunt Install
+      run: npm install -g grunt-cli
+    - name: Configure Couch
+      run: ./scripts/travis/couch-config
+    - name: Grunt CI-Compile
+      run: |
+        node --stack_size=10000 `which grunt` ci-compile
+    - name: Publish for testing
+      run: |
+        node --stack_size=10000 `which grunt` publish-for-testing
```
Comment (latin-panda) [inline .github/workflows/build.yml:60]: To double check, it's not necessary the default-config-tests, standard-config-tests, e2e-integration or  test-cht-default, right?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+      run: sudo apt-get install xsltproc
+    - name: Create logs directory
+      run: mkdir tests/logs
+    - name: npm CI
+      run: npm ci
+    - name: Grunt Install
+      run: npm install -g grunt-cli
+    - name: Horti Install
+      run: npm install -g horticulturalist
+    - name: Configure Couch
```
Comment (latin-panda) [inline .github/workflows/build.yml:109]: Do you need this here like in master? 
- name: Install medic-conf
      run:  npm install -g medic-conf
- name: Install pyxform
      run:  python -m pip install git+https://github.com/medic/pyxform.git@medic-conf-1.17#egg=pyxform-medic

**Review by latin-panda — COMMENTED** [review]
Nice! Just a couple of questions

File: .github/workflows/build.yml (hunk 1 above)
Comment (dianabarsan) [reply .github/workflows/build.yml:60]: Nope, in this version they are included in ci-compile: https://github.com/medic/cht-core/blob/7169-backport-to-310/Gruntfile.js#L1067

File: .github/workflows/build.yml (hunk 2 above)
Comment (dianabarsan) [reply .github/workflows/build.yml:109]: nope, we're not running medic-conf tests in this version.

Comment (dianabarsan) [reply .github/workflows/build.yml:109]: I just ported what was already running in travis config, no extras. to save time

**Review by latin-panda — APPROVED** [review]
LGTM! Thanks!!

**Comment by dianabarsan** [comment]
yay thanks!

**Pull request merged**